### PR TITLE
RTMD-74 - use hof-logger

### PR DESCRIPTION
**Pull request by joefitter**
* installed hof-logger
* removed bespoke logger
* attached logger to req in middleware
* removed logger from email service - this will use req.logger when moved

### CODE REVIEW

**Comment by JoeChapman** [comment]
You haven't published hof-logger yet

**Comment by joefitter** [comment]
I know

**Comment by joefitter** [comment]
it is waiting to be merged - this PR will pass when it is merged and published

**Comment by JoeChapman** [comment]
@joefitter rebase!

**Comment by JoeChapman** [comment]
Rebase @joefitter

**Comment by joefitter** [comment]
bah!

**Comment by joefitter** [comment]
done

File: services/email/index.js [hunk 1]
```diff
@@ -76,7 +75,6 @@ Emailer.prototype.send = function send(email, callback) {
 
     function sendCustomerEmail() {
       if (email.to) {
```
Comment (easternbloc) [inline services/email/index.js]: are we just not logging anymore here then?

**Comment by JoeChapman** [comment]
@joefitter not needed anymore because we are using bootstrap?

**Comment by joefitter** [comment]
yes and no - would be better for bootstrap to use hof-logger, think this was done at the same time as bootstrap

**Comment by joefitter** [comment]
saying that, this PR can be closed

**Pull request closed without merging**